info.json

```diff
@@ -0,0 +1,192 @@
+{
+    "name": "gcp-ca-service",
+    "version": "1.0.2",
+    "label": "GCP CA Service",
+    "description": "This connector integrates with GCP CA Service .",
+    "publisher": "Fortinet_CSE",
+    "icon_small_name": "cloudsec-png-small.png",
+    "icon_large_name": "cloudsec-png-large.png",
+    "category": [
+        "Utilities"
+    ],
+    "configuration": {
+        "fields": [
+            {
+                "title": "Upload Service Account JSON File",
+                "type": "file",
+                "name": "auth_file",
+                "required": true,
+                "visible": true,
+                "editable": true,
+                "tooltip": "The service account private key file."
+            }
+        ]
+    },
+    "operations": [
+        {
+            "operation": "list_certificate_authorities",
+            "title": "get_ca_crl",
+            "description": "get CA cert and CRL",
```

Comment:
Actions description should be more descriptive.
This is Applicable for all actions.